bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -17,6 +17,8 @@ spec:
         - mountPath: /var/log/kube-apiserver
           name: audit-dir
       command: ['/usr/bin/timeout', '105', '/bin/bash', '-ec'] # a bit more than 60s for graceful termination + 35s for minimum-termination-duration, 5s extra cri-o's graceful termination period
+      securityContext:
+        privileged: true
       args:
       - |
         echo -n "Fixing audit permissions."
```

Comment:
That said...let's move this bit into the MCO?  We can install a `systemd-tmpfiles` unit to do so.  Having an init container do this chmod seems strange at best.  I mean, we could just as easily do the `chmod` inside the go code too right?